pages/docs/troubleshooting/troubleshooting-communication.md

```diff
@@ -0,0 +1,44 @@
+---
+title: Problems with Establishing the Communication
```

Comment:
Let's convert this to "tip" alerts where appropriate + entries in the [FAQ](https://precice.discourse.group/tag/faq)